advisories/github-reviewed/2019/07/GHSA-x5rq-j2xg-h7qm/GHSA-x5rq-j2xg-h7qm.json

```diff
@@ -30,6 +30,25 @@
           ]
         }
       ]
+    },
+    {
+      "package": {
+        "ecosystem": "npm-amd",
```

Comment:
what is "npm-amd"?

Reply:
Thanks for catching that. The correct property is reflected in the most recent commit.